great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -382,20 +382,9 @@ def reconcile_profiler_variables(
         """
         effective_variables: ParameterContainer
         if variables and isinstance(variables, dict):
-            variables_configs: dict = self.variables.to_dict()["parameter_nodes"][
-                "variables"
-            ]["variables"]
-
-            if reconciliation_strategy == ReconciliationStrategy.NESTED_UPDATE:
-                variables_configs = nested_update(
-                    variables_configs,
-                    variables,
-                )
-            elif reconciliation_strategy == ReconciliationStrategy.REPLACE:
-                variables_configs = variables
-            elif reconciliation_strategy == ReconciliationStrategy.UPDATE:
-                variables_configs.update(variables)
-
+            variables_configs: dict = self.reconcile_profiler_variables_as_dict(
```

Comment:
@cdkini Personal preference, if you do not mind -- use "keyword style" (rather than positional) arguments.  Thanks!

Reply:
sure thing!

Reply:
resolved